src/routes/product.js

```diff
@@ -5,5 +5,6 @@ const productController = require('../controllers/product');
 const router = express.Router();
 
 router.post('/', productMiddleware, productController.addNewProduct);
+router.delete('/:id', productController.deleteProduct);
```

Comment:
Can we change the `:id` to `:productId` so the consumer doesn't enter the userId